### DISCUSSION TITLE
In progress images

### DISCUSSION THREAD

**Discussion by Xavman42** [Xavman42/GuitaRPG#7] [Show and tell]
![image](https://user-images.githubusercontent.com/79334532/227729590-7ca53be9-f087-4ea0-80dd-b4153f3a4cf8.png)
![image](https://user-images.githubusercontent.com/79334532/227729600-8247ed9d-4481-49f4-806d-25c0f8f62cd8.png)
![image](https://user-images.githubusercontent.com/79334532/227729602-8e61abf8-1a84-4218-9631-bcfc535a3626.png)
![image](https://user-images.githubusercontent.com/79334532/227729605-4da637e7-10bf-456b-8520-2a270c37cd90.png)
![image](https://user-images.githubusercontent.com/79334532/227729607-089218e8-71ca-45af-8662-1497e48abc2a.png)

**Comment by ajyoon** [comment]
very impressive, can't wait to hear it!
